lib/ansible/modules/monitoring/zabbix_host.py

```diff
@@ -239,6 +247,9 @@ def update_host(self, host_name, group_ids, status, host_id, interfaces, exist_i
                 parameters['proxy_hostid'] = proxy_id
             if visible_name:
                 parameters['name'] = visible_name
+            if description:
```

Comment:
I am curious, what was your test case? AFAICS it is only relevant if you like to clear an existing description. I don't know the api that well and if this is even supported, etc.